src/status_im2/contexts/onboarding/new_to_status/style.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im2.contexts.onboarding.new-to-status.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def full-screen {:flex 1 :position :relative})
```

Comment:
Why write `:position :relative` explicityly? That's the default

Reply:
I'm used to doing it, I do it every time I'm about to start an absolut position, that's because I'm more used to develop for web (the default position there is `static`). Also I couldn't find what's the default value for this attribute, do  you have any source? :sweat_smile: sometimes is not as easy to find info in the RN docs.

Solved in cae070c86054da83160ff4b825226848f49f1da6